src/sentry/middleware/integrations/parsers/gitlab.py

```diff
@@ -81,13 +81,20 @@ def get_response_from_gitlab_webhook(self):
 
         try:
             data = json.loads(self.request.body)
-        except ValueError as e:
-            logger.info("gitlab.body.parse_error", extra={"error": str(e)})
```

Comment:
This hasn't happened yet so it isn't a contributing factor to the incorrect mailbox sharding.